<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>588</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Rock, Paper, Scissors!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sciButton">
    <property name="geometry">
     <rect>
      <x>541</x>
      <y>-10</y>
      <width>275</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scissors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rockButton">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-10</y>
      <width>275</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paperButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>-10</y>
      <width>275</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Paper</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
